static-instrumentation/README.md

```diff
@@ -0,0 +1,17 @@
+# OpenTelemetry Java static instrumentation
+
+## Structure
+
+### Agent instrumenter
+
+Module enhancing OpenTelemetry Java Agent for static instrumentation. The modified agent is capable of instrumenting and saving a new JAR with all relevant instrumentations applied and necessary helper class-code included.
+
+In order to statically instrument a JAR, modified agent needs to be both attached (`-javaagent:`) and run as the main method (`io.opentelemetry.javaagent.StaticInstrumenter` class).
+
+### Gradle plugin
+
+Gradle plugin running static instrumentation agent during the `assemble` lifecycle task. The assembled archive contains statically instrumened class code.
+
+### Maven plugin
+
+Maven2 plugin running the static instrumentation agent during the `package` phase. Packaged archive contains statically instrumened class code.
```

Comment:
```suggestion
Maven plugin running the static instrumentation agent during the `package` phase. Packaged archive contains statically instrumened class code.
```

Well I assume it would work on Maven 3.x :)

Reply:
After rethinking this - M2 has been EOLed 8 years ago. Let's just use v3 API for the plugin and state M3 compatibility only.